packages/node-patches/src/subprocess.ts

```diff
@@ -47,7 +47,8 @@ fi
   }
 
   // fix execPath so folks use the proxy node
-  process.argv[0] = process.execPath = path.join(nodeDir, 'node');
+  process.argv[0] = process.execPath =
```

Comment:
In Windows, this will work only if `shell` is enabled when third parties `fork`or `spawn` a process.